### PR TITLE
Update PPP

### PR DESCRIPTION
**Pull request by JoelHo**

### CODE REVIEW

**Review by ZechariahTan — APPROVED** [review]
LGTM

File: docs/team/joelho.md [hunk 1]
```diff
@@ -3,38 +3,220 @@ layout: page
...
 * **Project management**:
+  * Setup team repo and organisation
+  * Designed icon and branding for product
+  * Ensured PRs and issues are linked to the correct author and milestone for proper tracking
   * Managed releases `v1.1` - `v1.4` (4 releases) on GitHub
 
 * **Enhancements to existing features**:
   * Augmented Person and Index to make them more testable (PR [\#57](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/57))
-  * Wrote additional tests for existing features to increase coverage by 1.63% (PR [\#57](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/57)), 2.56% (PR [\#78](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/78))
+  * Wrote additional tests for existing features to increase coverage by 1.63% (PR [\#57](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/57)), 2.56% (PR [\#78](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/78)),
```
Comment (chewterence) [inline docs/team/joelho.md:27]: that increased?

File: docs/team/joelho.md [hunk 2]
```diff
@@ -3,38 +3,220 @@ layout: page
...
+    2.81% (PR [\#286](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/286))
   * Refactored code using Java Optionals (PR [\#225](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/225))
   * Prevent Passengers from being deleted if they are being referenced by a Pool (PR [\#132](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/132))
+  * Fix Index being parsed incorrectly, resulting in wrong error message (PR [\#269](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/269))
   * Refactored the serialisation of Drivers and Pools such that they are stored as proper JSON (PR [\#125](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/125), [\#117](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/117))
   * Refactor toModelType of Passenger to better fit SLAP (PR [\#117](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/117))
+  * Add sanity checking for JSON read to ensure manually edited files have no errors (PR [\#256](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/256), [\#298](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/298))
```
Comment (chewterence) [inline docs/team/joelho.md]: should it be imperitive mood or past tense? added or add?

File: docs/team/joelho.md [hunk 3]
```diff
@@ -3,38 +3,220 @@ layout: page
...
 
+* **Community**:
+  * PRs reviewed (with non-trivial review comments): [\#16](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/16), [\#133](https://github.com/AY2021S2-CS2103T-W10-1/tp/pull/133)
+  * Substantial number of code related PRs merged, with code quality comments taken offline (and sadly not on github)
```
Comment (chewterence) [inline docs/team/joelho.md:38]: and sadly not on github?

Comment (chewterence) [reply docs/team/joelho.md:38]: did you mean substantial amounts

File: docs/team/joelho.md [hunk 4]
```diff
@@ -3,38 +3,220 @@ layout: page
...
+  1. Test case: `edit 7 n/Kelly`.<br>
+     Expected: Name of passenger previously named `Kristen Woo` is changed to `Kelly`. Status message shows all the details of `Kelly`. `Trip by Irfan Ibrahim` in Pool list updates to `Roy Balakrishnan, Kelly`.
+
+  1. Test case: `edit 7 d/TUESDAY`.<br>
+     Expected: No passenger is edited. Result box shows error: `TThe Passenger to be edited exists in a pool. Day cannot be edited.`. Command box text turns red.
+
+  1. Test case: `edit 7 t/1400`.<br>
+     Expected: Time of passenger named `Kelly` is changed to `1400`. Status message shows all the details of `Kelly`, and `NOTE: The passenger edited exists in a pool and has had their preferred trip time edited. This might result in a time difference of more than 15 minutes with the pool time.`
+
+--------------------------------------------------------------------------------------------------------------------
```
Comment (chewterence) [inline docs/team/joelho.md:222]: i did not add this, and its optional, so i have not reviewed this part of your PR

**Pull request merged**